firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -245,6 +248,14 @@ uint8_t moveStraight(uint8_t motor[], int endCol, int endRow)
   *currentMotorPos += (eighthStepsPerPulse * numSteps);
 
   return SUCCESS;
+
+  // Error code handling
```

Comment:
Though this is the correct way we want to do this, we're not going to do that in this PR. This PR mainly serves to give us the handler function. We're going to actually write the `moveDirect` and `moveAlongEdges` functions in a PR for issue #90. In other words, please remove the assignment on line 254 and the if statement starting on line 255.

Reply:
I just noticed this as well, but to clarify, these types of calls will be done in `moveDirect` and `moveAlongEdges`, not in either `moveStraight` or `moveDiagonal`